pennylane/__init__.py

```diff
@@ -65,7 +65,7 @@
     shadow_expval,
 )
 from pennylane.ops import *
-from pennylane.ops import adjoint, ctrl, exp, op_sum, pow, prod, s_prod
+from pennylane.ops import adjoint, ctrl, exp, op_sum, pow, prod, s_prod, ControlledQubitUnitary
```

Comment:
Do we really want to import this top-level?

Reply:
Yes, but maybe not right here. It is available as `qml.ControlledQubitUnitary` on master and should remain so following these changes. Moving it from `qubit.matrix_ops` broke using `qml.ControlledQubitUnitary`, and putting the import here seemed like the easiest way to make sure it was available. But maybe it would be better to put it elsewhere?

Reply:
I didn't know it was available as `qml.ControlledQubitUnitary`! Then we can leave it as it is.